src/docs/static-site-generation/debugging.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Debugging Static Site Generation in Stencil
+description: How to debug a pre-rendering or Static Site Generation step in Stencil
+url: /docs/static-site-generation-debugging
+contributors:
+  - mlynch
+---
+
+# Debugging Static Site Generation
+
+Static Site Generation, also known as pre-rendering, executes your components at build time to generate a snapshot of the rendered styles and markup to be efficiently served to search engines and users on first request.
+
+Since this step runs in a Node.js process instead of a browser, debugging can't be done directly in the browser. However, debugging is straightforward using existing Node.js debugging techniques.
+
+## Overview
+
+When pre-rendering, Stencil creates a file in `dist/prerender` that is used to actually execute your components. Debugging thi
```

Comment:
I believe you meant to delete 'Debugging thi'

Reply:
Thanks fixed